bindings/wasm/src/wallet.rs

```diff
@@ -86,16 +86,13 @@ pub async fn call_wallet_method_async(method: String, method_handler: &WalletMet
 #[wasm_bindgen(js_name = listenWalletAsync)]
 #[allow(clippy::await_holding_refcell_ref)]
 pub async fn listen_wallet(
-    vec: js_sys::Array,
+    vec: Vec<u8>,
```

Comment:
I am 0% sure about that, but I have no clue what to do about it

Reply:
Why did you change it?

Reply:
Because it's not an array of string now, it's an array of integers and I have absolutely no clue how to get them out

Reply:
I think `event_type.unwrap().as_f64().unwrap() as u8` should work, but since it's just u8 your change should be fine as well

Reply:
Yeah I thought about that but it seemed a bit weird, I'll do it